internal/services/search/search_service_resource.go

```diff
@@ -525,6 +555,10 @@ func resourceSearchServiceRead(d *pluginsdk.ResourceData, meta interface{}) erro
 				}
 			}
 
+			if props.SemanticSearch != nil {
+				semanticSearchSku = string(pointer.From(props.SemanticSearch))
+			}
```

Comment:
to map the None behaviour this can become:

```suggestion
			semanticSearchSku := ""
			if props.SemanticSearch != nil && pointer.From(props.SemanticSearch) != services.SearchSemanticSearchDisabled {
				semanticSearchSku = string(pointer.From(props.SemanticSearch))
			}
```

Reply:
Fixed.